src/Microsoft.Dnx.DesignTimeHost/Models/OutgoingMessages/DiagnosticsListMessage.cs

```diff
@@ -69,11 +78,11 @@ public override int GetHashCode()
             return base.GetHashCode();
         }
 
-        private class Comparer : IEqualityComparer<DiagnosticMessage>
+        private class Comparer : IEqualityComparer<DiagnosticMessageView>
```

Comment:
It compares lesser things than the `DiagnosticMessageView` (original `DiagnosticMessage`) `equals` method. Is there a reason to do so? Otherwise it is not needed.